pentaho-js/src/main/javascript/cdf/components/AnalyzerComponent.js

```diff
@@ -55,7 +65,13 @@ define([
       // process params and update options
       var d = this.dashboard;
       $.map(this.parameters, function(k) {
-        options[k[0]] = k.length == 3 ? k[2] : d.getParameterValue(k[1]);
+        options[k[0]] = k.length == 3 ? k[2] : d.getParameterValue(k[1]);    
+        if(myself.dateFormats[k[0]]) {
+          var formatedDate = moment(options[k[0]]).format(myself.dateFormats[k[0]]);
```

Comment:
Moment needs to be loaded using RequireJS. If it's available in the global scope a bug should be opened.